### PR TITLE
bb8: adding groupdisks into calculation - #2080

### PR DESCRIPTION
**Pull request by TomasJavurek**
bb8: adding groupdisks into calculation - #2080

### CODE REVIEW

File: lib/rucio/daemons/bb8/nuclei_background_rebalance.py [hunk 1]
```diff
@@ -27,14 +27,38 @@
...
+# groupdisks
+def group_space(site):
+    """
+    groupdisks of given site
+    contributing to primaries
+    """
+    site_groupdisks = []
+    group_total = 0
+    try:
+        site_groupdisks = parse_expression('site=%s&type=GROUPDISK' % site)
```
Comment (cserf) [inline lib/rucio/daemons/bb8/nuclei_background_rebalance.py]: Nope  :
```
rucio list-rses --expression "site=BNL-ATLAS&spacetoken=ATLASDATADISK"
BNL-OSG2_DATADISK
```
vs :
```
rucio list-rses --expression "site=BNL-ATLAS&type=GROUPDISK"
BNL-OSG2_DET-SLHC
BNL-OSG2_PHYS-HI
```
```
rucio list-rses --expression "site=BNL-ATLAS&spacetoken=ATLASGROUPDISK"
BNL-OSG2_DET-SLHC
BNL-OSG2_PHYS-HI
```

**Review by cserf — CHANGES_REQUESTED** [review]
Please have a look at the comment

File: lib/rucio/daemons/bb8/nuclei_background_rebalance.py (hunk 1 above)
Comment (TomasJavurek) [reply lib/rucio/daemons/bb8/nuclei_background_rebalance.py]: ok, but according to your example, there's no difference between spacetoken and type.

Comment (cserf) [reply lib/rucio/daemons/bb8/nuclei_background_rebalance.py]: Yes, but if you compare to :
```
rucio list-rses --expression "site=FZK-LCG2&spacetoken=ATLASDATADISK"
FZK-LCG2_PERF-EGAMMA
FZK-LCG2_DATADISK
FZK-LCG2_PERF-TAU                                                                                                                                                                                                                                                              
FZK-LCG2_PERF-IDTRACKING 
```
vs :

``` 
rucio list-rses --expression "site=FZK-LCG2&type=GROUPDISK"
FZK-LCG2_PERF-EGAMMA                                                                                                                                                                                                                                                           
FZK-LCG2_PERF-TAU                                                                                                                                                                                                                                                              
FZK-LCG2_PERF-IDTRACKING       
```

Comment (TomasJavurek) [reply lib/rucio/daemons/bb8/nuclei_background_rebalance.py]: this was done  Ithink

**Review by cserf — APPROVED** [review]

**Pull request merged**